opencanary.conf

```diff
@@ -42,9 +42,10 @@
                     "filename": "/var/tmp/opencanary.log"
                 }
             },
-            "per_service_logs": {
-                "ftp": "/var/tmp/ftp.log"
-            }
+            # Syntax to override per service logs:
+            # "per_service_logs": {
+            #     "ftp": "/var/tmp/ftp.log"
+            # }
```

Comment:
this broke the default `opencanary.conf` because there can't be an comments in javascript. i moved the explanation into the README: https://github.com/michelcrypt4d4mus/opencanary/commit/93b93317e0c8ea8be7559c77d6790810ee1cd2ed